doc/man3/EVP_DigestInit.pod

```diff
@@ -318,6 +321,27 @@ Similar to EVP_DigestFinalXOF() but allows multiple calls to be made to
 squeeze variable length output data.
 EVP_DigestFinalXOF() should not be called after this.
 
+=item EVP_DigestSerialize() and EVP_DigestDeserialize()
+
+EVP_DigestSerialize() serializes the state of the digest context I<ctx>
+and stores it in I<out>, unless I<out> is NULL.
+Otherwise the maximum necessary size of the output buffer is written to
+the I<outlen> parameter. If I<sig> is not NULL then before the call the
+I<outlen> parameter must contain the length of the I<out> buffer. If the
+call is successful the data is written to I<out> and the amount of data
+written to I<outlen>.
+
+EVP_DigestDeserialize() deserializes the data from I<in> of size I<inlen>
+into the digest context I<ctx>.
+
+These functions does not guarantee importability of state exported by a
```

Comment:
Need feedback from other @openssl/committers.
Is this ok? We dont allow incompatability across OpenSSL providers in most other places. Supporting a version number is not hard to do (I understand you are using it ephemerally, but is it possible that someone might want a persistent state?)